views/layouts/main.handlebars

```diff
@@ -10,13 +10,15 @@
   <meta name="defaultLanguage" content="en-US">
   <meta name="availableLanguages" content="en-US">
   <link rel="localization" href="/locales/send.{locale}.ftl">
+  <link rel="icon" type="image/png" href="/resources/favicon-32x32.png" sizes="32x32" />
   <script defer src="/l20n.min.js"></script>
 </head>
 <body>
   <header class="header"> 
     <div class="send-logo">
-      <img src="/resources/send_logo.svg" alt="Send"/>
-      <h1 class="site-title">Send</h1>
+      <a href="./">
```

Comment:
Nit: Can we just use "/" here?
Not sure if a leading zero will cause future issues if we ever do a nested directory structure for some reason.